packages/component-library/draft/Kaizen/GuidanceBlock/README.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Guidance Block
+navTitle: Guidance Block
+summaryParagraph: A Guidance Block provides a familiar layout that is visually prominent to draw attention to related content and encourage people to take an action. It guides people to new actions they haven't done before or connects them to relevant content and action they can take on another page.
+tags: ["Callout", "Callout card", "Media card", "Promos", "Campaign", "Notice", "Notification"]
+needToKnow:
+- Use a Guidance Block to encourage action on another page.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D20100%253A95" allowfullscreen></iframe>
+
+## Options
+
+To ensure the Guidance Block is distinct from other page content, you might use a different visual treatment for the card container.
+
+## To keep in mind
+
+The Guidance Block is a modular design element with a consistent structure and a clear call to action. It is commonly used to:
+
+* Prompt a user action (e.g. Select a focus area, create an action, redact comments, start onboarding)
+* Provide a summary view of longer form content contained on another page. This is similar to a content application of [tile](/components/tile) but with more prominence/hierarchy (e.g Manager learning)
+* Cross promote relevant content or functionality in a different part of the platform.
+
+### Anatomy
+
+* **Card container:** The Guidance Block content appears inside a card.
+* **Spot illustration:** Provide an illustration that assist the user in the task the Guidance Block describes.
+* **Title:** Use the title to introduce the action we want people to take.
+* **Description:** Use the description to give detail about what the user can expect about taking action.
+* **Action button:** Use a link styled as a primary action button (if there are no others on the page) or a default action button to navigate to the page with the action that the Guidance Block describes.
```

Comment:
It doesn't necessarily have to link to a page. The button can execute an action then render a notification saying things went well. For example, if we have a guidance block saying "We saw that you have a draft survey hanging for 5 weeks. Would you like to launch now? (LAUNCH)". This could do the action without any navigation.

Reply:
@juliohr I'm surprised by this. I would have thought the user would need more information about the draft survey in question to launch the survey than to launch it from the guidance block. If this Guidance Block implements an action immediately, while all others navigate somewhere, each might incorrectly set user expectations about how these work. Can you tell me more about that use case?

Reply:
Hmm, we have a use case where the action button doesn't navigate away, but rather launches a modal (as the context of the action doesn't require a full page form). 

@didoesdigital I hear what you're saying about user expectations. In this case I'm personally a bit more more concerned about how the the UX of safe exploration/forgiveness interacts with this.   Ideally the content and button labelling would do a good job of describing what selecting the button will do, but if that is difficult, or the action is not easily reversible then I'd expect at least a confirmation interaction to protect the user.

Reply:
Ah! I wonder if we then update the guidelines to say something like "at a minimum, it must have a confirmation modal"?

Reply:
I wonder if Lindsey's use case in comment reporting breaks this?